package_manifest.yaml

```diff
@@ -127,22 +127,15 @@ foreman_packages:
       - "{{ all_tags['foreman-el8'] }}"
     repoclosure_target_repos:
       el8:
-        - "el8-foreman-{{ foreman_version }}"
+        - "el8-foreman-{{ foreman_version }}-staging"
     repoclosure_lookaside_repos:
       el8:
         - el8-baseos
         - el8-appstream
         - el8-extras
         - el8-powertools
         - el8-puppet-7
-        - "el8-katello-candlepin-{{ candlepin_version }}"
-      rhel-8-x86_64:
-        - el8-baseos
-        - el8-appstream
-        - el8-extras
-        - el8-powertools
-        - el8-puppet-7
-        - "el8-katello-candlepin-{{ candlepin_version }}"
+        - "el8-katello-candlepin-{{ katello_version }}-staging"
```

Comment:
I think this should have been `candlepin_version`

Reply:
You right -- https://github.com/theforeman/foreman-packaging/pull/10171